app/scripts/modules/core/pipeline/config/stages/wait/SkipWait.tsx

```diff
@@ -15,7 +14,6 @@ interface IProps {
 }
 
 interface IState {
-  submitting: boolean;
```

Comment:
Getting rid of the `submitting` - we were getting into an issue where the component would unmount and the call to `setState` below would complain. Really, we don't need it, and it wasn't there before. I was trying to be too fancy when I refactored this yesterday.